packages/terra-framework-docs/src/terra-dev-site/doc/data-grid/FlowsheetDataGrid.4/About.5.doc.mdx

```diff
@@ -7,13 +7,76 @@ import FlowsheetDataGridPropsTable from 'terra-data-grid/src/FlowsheetDataGrid?d
 
 # Terra FlowsheetDataGrid
 
-## Getting Started
+## Table of Contents
+* [Overview](/components/cerner-terra-framework-docs/data-grid/flowsheet-data-grid/about#overview)
+* [Examples](/components/cerner-terra-framework-docs/data-grid/flowsheet-data-grid/about#examples)
+* [Accessibility](/components/cerner-terra-framework-docs/data-grid/flowsheet-data-grid/about#accessibility)
+* [Implementation](/components/cerner-terra-framework-docs/data-grid/flowsheet-data-grid/about#implementation)
+* [Properties](/components/cerner-terra-framework-docs/data-grid/flowsheet-data-grid/about#properties)
+* [Terra Standards](/components/cerner-terra-framework-docs/data-grid/flowsheet-data-grid/about#terra-standards)
 
+## Overview
+<Notice variant="important" ariaLevel="2">
+Due to poor support in Apple VoiceOver for spanned grid headings, the use of sections in the FlowsheetDataGrid is not currently recommended for applications used on Apple devices.
```

Comment:
In line 20, should FlowsheetDataGrid be Flowsheet Data Grid? I see it includes spacing in other parts of the file.

Reply:
@RachaelMarso - I can definitely add the spacing.  The name of the control is FlowsheetDataGrid.  However, the "Flowsheet Data Grid" with spaces is easier to read, I believe.